From 2530a5a6279a084c026e66d1473f2ab96ef35d1f Mon Sep 17 00:00:00 2001
From: gokul-khoros <gokulreko@gmail.com>
Date: Tue, 10 Nov 2020 22:38:40 +0530
Subject: [PATCH 03/11] git restore can be useful for restoring from staging
 area

---
 src/SecondApplication.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/SecondApplication.java b/src/SecondApplication.java
index 49aadaa..28ef76f 100644
--- a/src/SecondApplication.java
+++ b/src/SecondApplication.java
@@ -3,8 +3,8 @@ package src;
 public class SecondApplication {
 
     public String sideKick(){
-        String message = "hey how u dong";
-        return message;
+        String joeyTribbiani = "hey how u dong";
+        return joeyTribbiani;
     }
 
 }
-- 
2.23.0

