From 2d790cbd182856e688f983cd5c146eb01806d48e Mon Sep 17 00:00:00 2001
From: gokul-khoros <gokulreko@gmail.com>
Date: Wed, 11 Nov 2020 00:03:13 +0530
Subject: [PATCH 04/11] few notes are added

---
 src/SecondApplication.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/SecondApplication.java b/src/SecondApplication.java
index 28ef76f..4a0e1ff 100644
--- a/src/SecondApplication.java
+++ b/src/SecondApplication.java
@@ -3,6 +3,8 @@ package src;
 public class SecondApplication {
 
     public String sideKick(){
+        String learnt = "git log can be used to see all the commit history, those hash value can be used for checkout";
+        String phoebe = "ssh-keyegn along with rsa encrption can be used for authentication";
         String joeyTribbiani = "hey how u dong";
         return joeyTribbiani;
     }
-- 
2.23.0

