From 6b9bb52a95199e1328e4f2d701539c3470f9bd24 Mon Sep 17 00:00:00 2001
From: gokul-khoros <gokulreko@gmail.com>
Date: Wed, 11 Nov 2020 22:52:24 +0530
Subject: [PATCH 06/11] new commit for checking reset

---
 src/SecondApplication.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/SecondApplication.java b/src/SecondApplication.java
index 017e656..5001c8e 100644
--- a/src/SecondApplication.java
+++ b/src/SecondApplication.java
@@ -7,6 +7,7 @@ public class SecondApplication {
         String phoebe = "ssh-keyegn along with rsa encrption can be used for authentication";
         String joeyTribbiani = "hey how u dong";
 
+        int foodCount  = 100;
         return joeyTribbiani;
     }
 }
\ No newline at end of file
-- 
2.23.0

